src/PSRule.Rules.Azure/rules/Azure.MySQL.Rule.ps1

```diff
@@ -42,3 +42,23 @@ Rule 'Azure.MySQL.ServerName' -Ref 'AZR-000136' -Type 'Microsoft.DBforMySQL/serv
     # Can't start or end with a hyphen
     $Assert.Match($PSRule, 'TargetName', '^[a-z0-9]([a-z0-9-]*[a-z0-9]){2,62}$', $True);
 }
+
+# Synopsis: Azure Database for MySQL should store backups in a geo-redundant storage.
+Rule 'Azure.MySQL.GeoRedundantBackup' -Ref 'AZR-000323' -Type 'Microsoft.DBforMySQL/servers' -If { HasGeneralPurposeOrMemoryOptimizedTier } -Tag @{ release = 'GA'; ruleSet = '2022_12'; } {
+    $Assert.HasFieldValue($TargetObject, 'properties.storageProfile.geoRedundantBackup', 'Enabled').
+    Reason($LocalizedData.MySQLGeoRedundantBackupNotConfigured, $PSRule.TargetName).
+    PathPrefix('properties.storageProfile.geoRedundantBackup')
```

Comment:
You don't need to `PathPrefix` because `$Assert.HasFieldValue($TargetObject, 'properties.storageProfile.geoRedundantBackup', 'Enabled').` is already being used. The object path `properties.storageProfile.geoRedundantBackup` it will automatically be used

```suggestion
    $Assert.HasFieldValue($TargetObject, 'properties.storageProfile.geoRedundantBackup', 'Enabled').
    Reason($LocalizedData.MySQLGeoRedundantBackupNotConfigured, $PSRule.TargetName)
```